misc/python/materialize/cli/ci_logged_errors_detect.py

```diff
@@ -325,6 +328,15 @@ def get_error_logs(log_files: list[str]) -> list[ErrorLog]:
     return error_logs
 
 
+def sanitize_text(text: str, max_length: int = 4000) -> str:
+    if len(text) > max_length:
+        text = text[:max_length]
+
+    text = text.replace("```", r"\`\`\`")
```

Comment:
What if the text starts with a single `? I think that might cause some logic to change too.

Reply:
I added that to the demo to see what happens.